app/models/user_extension/authenticated_user.rb

```diff
@@ -74,15 +64,67 @@ def find_for_forget(email)
     # set to the currently logged in user.
     def current; Thread.current[:user]; end
     def current=(user); Thread.current[:user] = user; end
-  end
 
-  # Encrypts the password with the user salt
-  def encrypt(password)
-    self.class.encrypt(password, salt)
+    # DEPRECATED
+    # SHA1 hash the given password with the salt.
+    # we use has_secure_password now which uses bcrypt.
+    # but we still need sha1 for old passwords for which we use bcrypt(sha1)
+    def encrypt(password, salt)
```

Comment:
We're not using acts_as_authenticated anymore, right? So we can rename this any way we want.